apps/frontend-manage/src/components/groups/AddUserGroupMember.tsx

```
@@ -23,133 +20,132 @@ function AddUserGroupMember({
-        setOpen={() => setSuccessToast(false)}
-      />
-      <AddUserGroupErrorToast
-        open={errorToast}
-        setOpen={() => setErrorToast(false)}
-      />
-    </>
+            },
+          })
+
+          setSubmitting(false)
+          if (!!addedUser?.addUserToUserGroup) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)